New liaison document for WoT WG

egekorkan opened the pull request:
takes over #93 

With WoT IG not continuing, binding and use case mechanism being clear, willingness of WoT WG to give plugfest organization to CG, I have a leaner and nicer liaison document

egekorkan committed:
New liaison document for WoT WG

egekorkan committed:
Update liaisons.md

Co-authored-by: danielpeintner <daniel.peintner@gmail.com>

egekorkan committed:
Delete wot-ig-collaboration.md

egekorkan committed:
Revise liaisons.md for clarity and structure

Updated the document to clarify the roles and responsibilities of the W3C WoT Community Group liaisons. Enhanced the structure and wording for better readability.

egekorkan commented:
I have given it another pass. Some fixes are english and clarity. I have removed some unnecessary sections and made the document leaner. It should ready to review

egekorkan committed:
Update liaisons.md

Co-authored-by: danielpeintner <daniel.peintner@gmail.com>

ianbjacobs commented on liaisons.md:
```diff
@@ -1,3 +1,65 @@
+
+### Facilitating a Low Entry Barrier
+
+Because WG participation requires W3C membership, it is important to foster broader engagement and idea exchange on WoT-related topics.
```
I don't think it's necessary to refer to Membership here. The CG and the WG should be doing different things, and someone should get involved with either group depending on one's interests.

egekorkan replied:
Patent policy yes afaik. Charter can also happen, never seen it yet. Of course WG Charter and WG-level patent policy

ianbjacobs replied:
Working Groups should not accept contributions from non-participants unless the contributors make licensing commitments. But, in addition, if a CG participant contributes something to a CG, they are making the same RF licensing commitment as they would be making in the WG, and if the work is taken up by the WG, their commitment follows it.

ianbjacobs commented on liaisons.md:
```diff
@@ -1,3 +1,65 @@
+### Bindings
+
+The CG provides a venue to develop bindings for contributors who cannot join the WG.
+This is relevant when the _owner_ of a protocol or the WG is not willing or able to create a binding for a protocol of interest to a CG participant.
```
I think you can delete this sentence.

egekorkan replied:
```suggestion
```

ianbjacobs commented on liaisons.md:
```diff
@@ -1,3 +1,65 @@
+The CG provides a venue to develop bindings for contributors who cannot join the WG.
+This is relevant when the _owner_ of a protocol or the WG is not willing or able to create a binding for a protocol of interest to a CG participant.
+This enables collaboration and supports the creation of public, stable URIs needed for registry submissions.
+By no means, the development of bindings is limited to CG participants.
```
If you want to encourage people to do bindings in the CG, then I think it's useful to say "Here are the good reasons why you benefit from incubating bindings in the CG." (But that would be independent of deleting the current sentence.)

egekorkan replied:
But once an entry is in the registry, it may not need any further standardization (or any other commits to the spec text). The only requirement for going from initial to current (stable) is doing testing, which lives in a separate document. Thus, from my point of view, it is not comparable to a specification being taken up in a WG as a FPWD.

egekorkan commented on liaisons.md:
```diff
@@ -1,3 +1,65 @@
+
+The following requirements are mandated by external documents:
+
+- The CG bindings must adhere to the [W3C CLA](https://www.w3.org/community/about/process/cla/) that each CG participant agrees to upon joining the CG.
```
> Is a binding a specification?

Definitely. It is giving specific guidance on how a well-known protocol should be implemented in the context of WoT. However, by the rules of the registry, it does not have to be a Rec Track document to be accepted to the registry.

egekorkan committed:
Update liaisons.md

Co-authored-by: Ted Thibodeau Jr <tthibodeau@openlinksw.com>

egekorkan committed:
Update liaisons.md

egekorkan commented on liaisons.md:
```diff
@@ -1,3 +1,65 @@
+
+### Bindings
+
+The CG provides a venue to develop bindings for contributors who cannot join the WG.
```
I will not commit this suggestion as it gives the idea that CG is for the incubation of the bindings and that they are not usable until they are in WG. From the registry's perspective (see https://github.com/w3c/wot-cg/pull/227#discussion_r2661103552), there is no difference between entries' trustworthiness.

egekorkan committed:
Update liaisons.md

egekorkan committed:
Update liaisons.md

egekorkan committed:
Update CG binding process details in liaisons.md

Clarified the role of CG in developing bindings and their submission process to the Web of Things Binding Registry.

egekorkan commented on liaisons.md:
```diff
@@ -1,3 +1,65 @@
+
+The CG provides a venue to develop bindings for contributors who cannot join the WG.
+This is relevant when the _owner_ of a protocol or the WG is not willing or able to create a binding for a protocol of interest to a CG participant.
+This enables collaboration and supports the creation of public, stable URIs needed for registry submissions.
```
I think this is resolved now. Please have another look.

egekorkan committed:
Update liaisons.md

Co-authored-by: Ted Thibodeau Jr <tthibodeau@openlinksw.com>
